examples/hybrid/sphere/balanced_flow_rhoe.jl

```diff
@@ -1,25 +1,20 @@
 using Test
 using ClimaCorePlots, Plots
-using DiffEqCallbacks
-using JLD2
 
 include("baroclinic_wave_utilities.jl")
 
-jld2_callback = PeriodicCallback(output_writer, 864000; initial_affect = true)
+const sponge = false
```

Comment:
Let's add 
```
#########################
# set your experiment specs here
#########################
```
so that people have a clear idea of where to specify their prefered experiment setups

Reply:
Sure. I can add that. Though, I'm unclear how it clarifies things? What else can you do in a test case file other than set experiment specs?

Reply:
For example, I would rather not touch the anything below this section (intial condition, tendency set up, etc.) and make it clear that everything you want to specify for this experiment is in this section.

Reply:
Okie dokie, I'll add a comment.